features/loading-lazy.yml

```diff
@@ -2,4 +2,5 @@ name: Lazy-loading images and iframes
 description: |
   The `loading="lazy"` attribute for `<img>` and `<iframe>` elements blocks loading the external resource until the user scrolls to that element's part of the page.
 spec: https://html.spec.whatwg.org/multipage/urls-and-fetching.html#lazy-loading-attributes
+group: scrolling
```

Comment:
There is a connection to scrolling with this feature, but I think it's primarily a loading performance feature. If we had a primary group for that I think it'd be OK to also list it in the scrolling group, but to have it as only scrolling feels odd to me. WDYT?

Reply:
Agreed, I was on the fence with this one. Ok to remove.